lib/parsers/exif_parser.rb

```diff
@@ -175,7 +175,7 @@ def method_missing(*a)
   def exif_from_tiff_io(constrained_io, should_include_sub_ifds = false)
     Measurometer.instrument('format_parser.exif_parser.exif_from_tiff_io') do
       extended_io = IOExt.new(constrained_io)
-      exif_raw_data = EXIFR::TIFF.new(extended_io)
+      exif_raw_data = EXIFR::TIFF.new(extended_io, load_thumbnails: false)
```

Comment:
One question: could it be useful to provide this as a config for the library? I can imagine some might actually need those thumbnails.

Reply:
You mean via `intrinsics`? That is a good point. I'll change this. 
On this: are the intrinssics something we want to maintaining stable across multiple versions?

Reply:
Not necessarily, just a way to configure this when using the library.

On the intrinsics, I think it would be nice, though I wouldn't consider it as a priority.